README.md

```diff
@@ -374,6 +374,25 @@ FactoryHelper::Time.forward(23, :morning) # => "2014-09-26 06:54:47 -0700"
 FactoryHelper::Time.backward(14, :evening) #=> "2014-09-17 19:56:33 -0700"
 ```
 
+###FactoryHelper::Timestamps
+---------------------
+Generates random time between times up to the nanosecond level (that is 1_000_000_000 of a second)
+
+```ruby
+# Random time between times
+FactoryHelper::Timestamps.between(from, to) #=> "2012-05-05 05:05:31 +0300"
+
+# Random time in the future (up to maximum of N seconds)
```

Comment:
Consider revising to e.g. "Random time up to N seconds in the future" & "... in the past"